src/components/header.jsx

```
@@ -1,21 +1,116 @@
+import { useEffect, useState } from "react";
+import { Link, useSearchParams } from "react-router-dom";
+import {
+  SignedIn,
+  SignedOut,
+  UserButton,
+  SignIn,
+  useUser,
+} from "@clerk/clerk-react";
 import { Button } from "./ui/button";
-
+import { BriefcaseBusiness, Heart, PenBox } from "lucide-react";
```

ESLint

[error] 'Heart' is defined but never used.

(no-unused-vars)